apps/sensenet/src/components/AddButton.tsx

```diff
@@ -212,7 +212,7 @@ export const AddButton: React.FunctionComponent<AddButtonProps> = props => {
                 style={{ padding: '10px 0 10px 10px' }}
                 onClick={() => {
                   setShowSelectType(false)
-                  openDialog({ name: 'add', props: { schema: childType, parent } })
+                  openDialog({ name: 'add', props: { schema: childType, parentPath: props.path } })
```

Comment:
Shouldn't this be props.parent.Path? When I select the it workspace under content and try to create a folder under it now it creates under /Content

Reply:
When there was no selected element, props.parent was undefined so I changed the value of this prop now to a statement to handle both situations (with or without actual parent).